src/pages/products-carousel/products-carousel.js

```diff
@@ -47,24 +50,30 @@ const ProductsCorousel = ({ category }) => {
     );
   }
 
-  const handleClick = model => {
+  const handleClick = (model) => {
     dispatch(setModelsFilter([model.name[1].value]));
     dispatch(setPatternsFilter([]));
     dispatch(setColorsFilter([]));
     dispatch(setSearchFilter(''));
     dispatch(setHotItemFilter(false));
     dispatch(
       setPriceFilter([
-        Math.min(...filterData.map(product => product.basePrice[0].value)),
-        Math.max(...filterData.map(product => product.basePrice[0].value))
+        Math.min(...filterData.map((product) => product.basePrice[0].value)),
+        Math.max(...filterData.map((product) => product.basePrice[0].value))
       ])
     );
   };
 
   return (
     <div className={styles.container}>
-      <AwesomeSlider className={styles.slider} mobileTouch>
-        {models.map(model => (
+      <AutoplaySlider
+        play={true}
+        cancelOnInteraction={false}
+        interval={4000}
```

Comment:
Please, move countdown into the config file

Reply:
fixed